frame/session/src/historical.rs

```diff
@@ -385,7 +370,7 @@ mod tests {
 
 			assert_eq!(StoredRange::get(), Some((0, 100)));
```

Comment:
We started session `98`, but `100` is in our stored range? what information do we have available about session 100?

Reply:
we have stored range `[0; 100)` and because we delay by one session in this test runtime, the validator of next session is available. so we have information of session 99 when we start session 98